src/Components/CreateImageWizardV2/steps/Repositories/components/BulkSelect.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  DropdownToggleCheckbox,
+} from '@patternfly/react-core/deprecated';
+
+import { ApiRepositoryResponseRead } from '../../../../../store/contentSourcesApi';
+
+interface BulkSelectProps {
+  selected: Set<string>;
+  contentList: ApiRepositoryResponseRead[];
+  deselectAll: () => void;
+  perPage: number;
+  handleAddRemove: (
+    repo: ApiRepositoryResponseRead | ApiRepositoryResponseRead[],
+    selected: boolean
+  ) => void;
+  isDisabled: boolean;
+}
+
+export function BulkSelect({
```

Comment:
When removing the page of repos via bulk select the remove modal gets triggered even for new blueprint

![repos-modal](https://github.com/osbuild/image-builder-frontend/assets/49452678/ea849ed6-a1a5-4a1c-9712-0987f7021aef)

Reply:
Hmm will fix this.